src/main/java/seedu/address/model/recipe/Step.java

```diff
@@ -25,10 +24,10 @@ public Step(String step) {
     }
 
     /**
-     * Returns true if the given list of steps is not empty.
+     * Returns true if the given step is not empty.
      */
     public static boolean isValidSteps(String test) {
```

Comment:
More accurate fn name since fn only checks current step:

```suggestion
    public static boolean isValidStep(String test) {
```